### ISSUE TITLE
failed to create the raw output file C:/Users/jason_06tt1g7/coding-public/newBox/ubuntu-bionic-18.04-cloudimg-console.log (VERR_PATH_NOT_FOUND)

### ISSUE THREAD

**Issue by websafe** [reddingwebpro/cognacbox#2]
Hello there,

I'm running `Virtualbox  6.1.4 r136177 (Qt5.6.2)` on Windows 10.

My `Vagrantfile`:
```
Vagrant.configure("2") do |config|

    config.vm.box = "reddingwebpro/cognacbox"
    config.vm.hostname = "cognacbox"
    config.vm.network "forwarded_port", guest: 80, host: 8080
    config.vm.network "private_network", ip: "192.168.33.10"
    config.vm.synced_folder ".", "/var/www", :mount_options => ["dmode=777", "fmode=777"]
    config.vm.provider "virtualbox" do |v|
        v.memory = 4096
        v.cpus = 4
    end
end
```

After running `vagrant up` I get the following result:
```
Bringing machine 'default' up with 'virtualbox' provider...
==> default: Checking if box 'reddingwebpro/cognacbox' version '2.0' is up to date...
==> default: Clearing any previously set forwarded ports...
==> default: Clearing any previously set network interfaces...
==> default: Preparing network interfaces based on configuration...
    default: Adapter 1: nat
    default: Adapter 2: hostonly
==> default: Forwarding ports...
    default: 80 (guest) => 8080 (host) (adapter 1)
    default: 22 (guest) => 2222 (host) (adapter 1)
==> default: Running 'pre-boot' VM customizations...
==> default: Booting VM...
There was an error while executing `VBoxManage`, a CLI used by Vagrant
for controlling VirtualBox. The command and stderr is shown below.

Command: ["startvm", "62e7dda6-4adf-43a4-b721-0d8189daeea8", "--type", "headless"]

Stderr: VBoxManage.exe: error: RawFile#0 failed to create the raw output file C:/Users/jason_06tt1g7/coding-public/newBox/ubuntu-bionic-18.04-cloudimg-console.log (VERR_PATH_NOT_FOUND)
VBoxManage.exe: error: Details: code E_FAIL (0x80004005), component ConsoleWrap, interface IConsole
```

The machine is not starting. Is there a known solution?

I gues the path **C:/Users/jason_06tt1g7/coding-public/newBox/** is causing the problem?

**Comment by reddingwebpro** [comment]
Yes, looks like a bug in the build, I'll rebuild the vagrant file and post it shortly.
________________________________

**Comment by liangzhong** [comment]
Any update? @reddingwebpro 
Thank you.

**Comment by xolge** [comment]
I second this issue ( i have the same issue).
```
There was an error while executing `VBoxManage`, a CLI used by Vagrant
for controlling VirtualBox. The command and stderr is shown below.

Command: ["startvm", "03f83b91-97db-4680-b1f4-9332d61cecc2", "--type", "headless"]

Stderr: VBoxManage: error: RawFile#0 failed to create the raw output file C:/Users/jason_06tt1g7/coding-public/newBox/ubuntu-bionic-18.04-cloudimg-console.log (VERR_FILE_NOT_FOUND)
VBoxManage: error: Details: code NS_ERROR_FAILURE (0x80004005), component ConsoleWrap, interface IConsole
```

Thank you for updating this; i await its release. :)

**Comment by wucherpfennig** [comment]
same question

**Comment by Franky1** [comment]
same error here

**Comment by wucherpfennig** [comment]
for the others maybe this helps: https://github.com/reddingwebpro/cognacbox/issues/3#issuecomment-642915505

**Comment by reddingwebpro** [comment]
Testing a fix for this now. Hoping to release it tomorrow.

**Comment by reddingwebpro** [comment]
Source of the problems seems to be related to https://github.com/hashicorp/vagrant/issues/10578 -- implementing a patch in the code, and should be available shortly.

**Comment by reddingwebpro** [comment]
Fixed in v2.2 -- updated master.
